eng/pipelines/templates/steps/use-python-version.yml

```diff
@@ -0,0 +1,27 @@
+parameters:
+  versionSpec: ''
+
+steps:
+  # use python 3.8 for tooling. packaging. platform.
+  - task: UsePythonVersion@0
+    displayName: "Use Python 3.8"
+    inputs:
+      versionSpec: 3.8
```

Comment:
3.8?

Reply:
I am reusing a work around approach Scott is using to install 3.10 in python pipeline by running a script that pulls 3.10 image and install

Reply:
3.8 is only used to run the script which install 3.10